docs/CI.md

```diff
@@ -0,0 +1,49 @@
+# Continuous Integration (CI)
+
+Before a commit to Sulong is merged, it first has to pass all our CI tests.
+We have two checks in place:
+
+1. The OCA check: In order to contribute to Sulong, [each contributor has
+to sign and submit the Oracle Contributor Agreement (OCA)](CONTRIBUTING.md).
+The OCA check confirms that a contributor has signed the OCA.
+
+2. The Travis gate: We run a test gate in which we execute our test cases
+and static analysis tools.
+
+## The Travis Gate
+
+Travis is an open-source CI service. We use this service to execute our
+test gate. The gate checks are defined in the
+[`.travis.yml` configuration file](/.travis.yml).
+
+The gate performs the following tasks:
+
+1. a check for the warning-free compilation of all Java sources with
+   `javac` and `ECJ`: To check the error-free compilation locally, use
+   `mx build` with the `JDT` environment variable set to an `ECJ` jar
+   file to compile with `ECJ` and otherwise with `javac`.
+2. the execution of all [test suites](docs/TESTS.md): Use `mx su-tests` to
+   execute the test cases.
+3. confirming that no Java files are formatted by `eclipseformat`
+   (which is normally executed when saving a Java file in Eclipse):
+   Use `mx eclipseformat --primary` with the `ECLIPSE_EXE` environment
+   variable set to eclipse.
+4. confirming that there are no Java style violations using `checkstyle`
+   (the configurations are defined per-project in the `checkstyle` attribute
+   of the [suite.py](/mx.sulong/suite.py) file). Use `mx checkstyle` to
+   execute the check locally.
+5. confirming that there are no Python format or style violations using
+   `pylint`. Use `mx pylint` to execute the check locally.
+6. confirming that no C or C++ files are formatted by using `clang-format`:
```

Comment:
Saying 'no C or C++ files are formatted' sounds like you're checking that they don't pass the test. How about 'confirming that C and C++ files are correctly formatting using clang-format'.?